onadata/libs/serializers/organization_serializer.py

```diff
@@ -73,6 +73,10 @@ def update(self, instance, validated_data):
             instance.user.last_name = last_name
             instance.user.save()
 
+        if "email" in validated_data:
+            instance.user.email = validated_data.pop("email")
+            instance.user.save()
```

Comment:
We can avoid saving twice by taking `instance.user.save()` out of the `if` block and removing `instance.user.save()` in `if` block above it.